frontend/src/components/pages/CampCreation/index.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from "react";
+import { Box, Button } from "@chakra-ui/react";
+import CampDetails from "./CampDetails";
+import ScheduleSessions from "./ScheduleSessions";
+import RegistrationForm from "./RegistrationForm";
+
+enum CampCreationPages {
+  CampDetailsPage,
+  ScheduleSessionsPage,
+  RegistrationFormPage,
+}
+
+const CampCreationPage = (): JSX.Element => {
+  const [currentPage, setCurrentPage] = useState<CampCreationPages>(
+    CampCreationPages.CampDetailsPage,
+  );
+
+  const getPageComponent = (nextPage: CampCreationPages) => {
+    // will also need logic blocking pages if current page not complete
+    switch (nextPage) {
+      case CampCreationPages.CampDetailsPage:
+        return <CampDetails />;
+      case CampCreationPages.ScheduleSessionsPage:
+        return <ScheduleSessions />;
+      case CampCreationPages.RegistrationFormPage:
+        return <RegistrationForm />;
+      default:
+        throw new Error("never reached");
+    }
+  };
+
+  return (
+    <Box>
```

Comment:
You could perhaps consider adding some margins here, but if it's unneccesary for a dummy component that's fine too.